src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/PCCRegion.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import javax.persistence.*
```

Comment:
heads up - i think this may fail the ktlint source checks

Reply:
will update